packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts

```diff
@@ -206,11 +206,7 @@ export default class MeasurementReport {
 
         const _meta = MeasurementReport.generateDatasetMeta();
 
-        const derivationSourceDataset = {
-            ...instance,
-            _meta: _meta,
-            _vrMap: _vrMap
-        };
+        const derivationSourceDataset = copyStudyTags(instance);
```

Comment:
Use the new copyStudyTags which encapsulates which tags are needed.  The previous code was creating invalid DICOM because it would copy things like Rows and Columns, and wouldn't work on the multiframe inheritted objects.